test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5984,6 +5984,42 @@ public void PackCommand_PackIcon_MissingIconFile_Fails()
             TestPackIconFailure(testDirBuilder, NuGetLogCode.NU5019.ToString());
         }
 
+        [Theory]
+        [InlineData(".jpg", null)]
+        [InlineData(".png", null)]
+        [InlineData(".PnG", null)]
+        [InlineData(".jpeg", null)]
+        [InlineData(".JpEg", null)]
+        [InlineData("abc.JpEg", null)]
+        [InlineData(".x", NuGetLogCode.NU5045)]
+        [InlineData(".jpeg.x", NuGetLogCode.NU5045)]
+        [InlineData("", NuGetLogCode.NU5044)]
+        public void PackCommand_PackIcon_InvalidExtension_SucceedOrFail(string fileExtension, NuGetLogCode? logCode)
```

Comment:
This appears to be an almost exact duplicate of the unit test you added in `NuGet.Packaging.Test`. I don't think we should have duplicated scenario tests, particularly in slow functional/integration tests. `NuGet.CommandLine.Test` should be testing integration of components, but leave the exhaustive tests to fast, unit tests.

Reply:
Yeah, it's almost the same test. I'm taking a 'just in case' approach and test whenever you can. How should test the integration of the icon file extension validation? I wrote a test here and not int dotnet/msbuild.integration test project to avoid test time overhead.

Reply:
Removed test to avoid redundancy.